pkg/rb2wf/converter_test.go

```diff
@@ -0,0 +1,339 @@
+										"raw":            "{{params.an_integer}}",
+									},
+								},
+							},
+							Resources: map[string]string{
+								"db_slug": "dbID",
+							},
+						},
+					},
+				},
+				{
+					ID:             "restBlockID",
+					Slug:           "rest_block_id",
+					BlockKind:      "stdapi",
+					StartCondition: "\"hello\" === params.test_param",
+					BlockKindConfig: api.BlockKindConfig{
+						StdAPI: &api.BlockKindConfigStdAPI{
+							Namespace: "rest",
+							Name:      "request",
+							Request: map[string]interface{}{
+								"headers": map[string]interface{}{
+									"header1": map[string]interface{}{
+										"__airplaneType": "template",
+										"raw":            "header2",
+									},
+								},
+								"method": "GET",
+								"path": map[string]interface{}{
+									"__airplaneType": "template",
+									"raw":            "/heathz",
+								},
+								"urlParams": map[string]interface{}{
+									"test1": map[string]interface{}{
+										"__airplaneType": "template",
+										"raw":            "test2",
+									},
+								},
+							},
+							Resources: map[string]string{
+								"rest_slug": "restID",
+							},
+						},
+					},
+				},
+				{
+					ID:        "emailBlockID",
+					Slug:      "email_block_id",
+					BlockKind: "stdapi",
+					BlockKindConfig: api.BlockKindConfig{
+						StdAPI: &api.BlockKindConfigStdAPI{
+							Namespace: "email",
+							Name:      "message",
+							Request: map[string]interface{}{
+								"message": map[string]interface{}{
+									"__airplaneType": "template",
+									"raw":            "This is a message!",
+								},
+								"recipients": []map[string]interface{}{
+									{
+										"email": "bob@example.com",
+										"name":  "Bob",
+									},
+								},
+								"sender": map[string]interface{}{
+									"email": map[string]interface{}{
+										"__airplaneType": "template",
+										"raw":            "yolken@airplane.dev",
+									},
+									"name": map[string]interface{}{
+										"__airplaneType": "template",
+										"raw":            "BHY",
+									},
+								},
+								"subject": map[string]interface{}{
+									"__airplaneType": "template",
+									"raw":            "Hello",
+								},
+							},
+							Resources: map[string]string{
+								"email_slug": "emailID",
+							},
+						},
+					},
+				},
+				{
+					ID:        "slackBlockID",
+					Slug:      "slack_block_id",
+					BlockKind: "stdapi",
+					BlockKindConfig: api.BlockKindConfig{
+						StdAPI: &api.BlockKindConfigStdAPI{
+							Namespace: "slack",
+							Name:      "message",
+							Request: map[string]interface{}{
+								"channelName": "notif-deploys-test",
+								"message": map[string]interface{}{
+									"__airplaneType": "template",
+									"raw":            "Hello!",
+								},
+							},
+						},
+					},
+				},
+				{
+					ID:        "formBlockID",
+					Slug:      "form_block_id",
```

Comment:
some of these slugs are suffixed with `_id`

```suggestion
					Slug:      "form_block_slug",
```

Reply:
Fixed in https://github.com/airplanedev/cli/pull/823.